company-keywords.el

```diff
@@ -297,6 +297,26 @@
     (enh-ruby-mode . ruby-mode))
   "Alist mapping major-modes to sorted keywords for `company-keywords'.")
 
+(with-eval-after-load 'make-mode
+  (require 'seq)
+  (mapc
+   (lambda (mode-stmnts)
+     (setf (alist-get (car mode-stmnts) company-keywords-alist)
+           (seq-uniq
+            (append
+             makefile-statements
+             makefile-special-targets-list
+             (mapcan #'identity
```

Comment:
Might also want to pass it through `cl-delete-duplicates`.

Reply:
I think `seq-uniq` (line 305) did the same thing.

Reply:
Not sure how I missed that. Thanks.